IntegrationTest/IntegrationTest.csproj

```diff
@@ -76,9 +76,6 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <None Include="Configuration\config.json">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
```

Comment:
This seems... counter-intuitive? It looks like you're copying the default config to the config.json but then not copying it to the output directory?